common/helpers/field/add-errorlist-to-errors.js

```diff
@@ -9,7 +9,7 @@ function addErrorListToErrors(errors = {}, fields) {
         ...field,
         ...error,
       }),
-      href: `#${field.id}`,
```

Comment:
If field.id is always a string or undefined, we could just have `href: field.id` here

Reply:
Unfortunately not, we need the `#` at the start of the string to represent the link to the element.

Reply:
Oops, I missed the `#`, carry on :D